src/compiler/fuzzer.rs

```diff
@@ -0,0 +1,795 @@
+use num_bigint::{BigInt, ToBigInt};
+use num_traits::ToPrimitive;
```

Comment:
I think it would be good to fix the clippy warnings in this file, and other new files.
(and, separately, make CI fail if there are clippy warnings)

Reply:
it passed the previous clippy.  i'll see if it passes the current clippy.  i have been fixing all clippy issues that come up.